packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/07-publication/02-metadata.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Plugin Metadata
+---
+
+## Plugin Metadata Specification
+
+The plugin metadata is necessary to allow the App frontend to interact with generic plugins:
+
+- Now: generic setup (installation, update, uninstallation)
+  - Allows the frontend to render the necessary fields for the input being required to setup the plugin (e.g., the list of initial members of the Multisig plugin)
+- Future: render an UI in a generic way (buttons, text fields, flows) within the specs of the Open Design System (ODS) (e.g. manage the list of Multisig members or the approval settings)
+
+Currently, two kinds of metadata exist:
+
+1. Release metadata
+2. Build metadata
+
+### Release Metadata
+
+The release metadata is a `.json` file stored on IPFS with its IPFS CID published for each release in the [PluginRepo](../../../01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/index.md) (see also the section about [versioning](../07-publication/01-versioning.md/#)).
+
+The intention is to provide an appealing overview of each releases functionality.
+It can be updated with each call to [`createVersion()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-createversion) in `IPluginRepo` by the repo maintainer
+It can be replaced at any time with [`updateReleaseMetadata()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-updatereleasemetadata) in `IPluginRepo` by the repo maintainer
+
+The `release-metadata.json` file consists of the following entries:
+
+| Key         | Type        | Description                                                                  |
+| ----------- | ----------- | ---------------------------------------------------------------------------- |
+| name        | `string`    | Name of the plugin (e.g. `"Multisig"`)                                       |
+| description | `string`    | Description of the plugin release and its functionality.                     |
+| images      | UNSPECIFIED | Optional. Contains a series of images advertising the plugins functionality. |
+
+#### Example
+
+```json
+{
+  "name": "Multisig",
+  "description": "",
+  "images": {}
+}
+```
+
+### Build Metadata
+
+The build metadata is a `.json` file stored on IPFS with its IPFS CID published for each build **only once** in the [PluginRepo](../../../01-how-it-works/02-framework/02-plugin-management/01-plugin-repo/index.md) (see also the section about [versioning](../07-publication/01-versioning.md/#)).
+
+The intention is to inform about the changes that were introduced in this build compared to the previous one and give instructions to the App frontend and other users on how to interact with the plugin setup and implementation contract.
+It can be published **only once** with the call to [`createVersion()`](../../../03-reference-guide/framework/plugin/repo/IPluginRepo.md/#external-function-createversion) in `IPluginRepo` by the repo maintainer
```

Comment:
Mising ending period.

Reply:
Changed in defce3f5d16cac6b573476b5c5a68d86f26597b6